interfaces.go

```diff
@@ -263,7 +270,9 @@ type QueryExecutor interface {
 	ExecuteWithOptions(opts Options, stmt string, params ...interface{}) error
 	// Execute executes a DML query
 	Execute(stmt string, params ...interface{}) error
-	// ExecuteAtomically executs multiple DML queries with a logged batch
+	// ExecuteAtomically executes multiple DML queries with a logged batch, and takes context
+	ExecuteAtomicallyWithOptions(opts Options, stmt []string, params [][]interface{}) error
```

Comment:
Could we move this to after the `ExecuteAtomically` so the _WithX_ func is 2nd? 🙂